nipype/interfaces/dcm2nii.py

```diff
@@ -399,7 +414,7 @@ def _format_arg(self, opt, spec, val):
                 spec.argstr += ' n'
                 val = True
         if opt == 'source_names':
-            return spec.argstr % val[0]
+            return spec.argstr % (os.path.dirname(val[0]) or '.')
```

Comment:
since the dicoms are copied to the node, couldn't this just be `'.'`

Reply:
not when used as an interface.